index.js

```diff
@@ -105,7 +105,11 @@ class KintoServer {
 
   killAll() {
     return new Promise(resolve => {
-      spawn("killall", ["pserve"]).on("close", () => resolve());
+      if (process.platform === "win32") {
```

Comment:
will it work with win64?

Reply:
The only possible Windows value [is `win32`](https://nodejs.org/api/process.html#process_process_platform).